src/Campaigns/Routes/PublishCampaign.php

```diff
@@ -0,0 +1,73 @@
+<?php
+
+namespace Give\Campaigns\Routes;
+
+use Give\API\RestRoute;
+use Give\Campaigns\Models\Campaign;
+use Give\Campaigns\Routes\Traits\RestResponses;
+use Give\Campaigns\ValueObjects\CampaignStatus;
+use Give\Framework\Exceptions\Primitives\Exception;
+use WP_REST_Request;
+use WP_REST_Response;
+
+/**
+ * @unreleased
+ */
+class PublishCampaign implements RestRoute
+{
+    use RestResponses;
+
+    /** @var string */
+    protected $endpoint = 'campaigns/publish/(?P<id>[0-9]+)';
```

Comment:
This endpoint should probably be `campaigns/{id}/publish`, which would more closely match a RESTful resource.

Reply:
Resolved in d362f6f69afe0305a59fe6da75ee94c004dfda86 and 849de72a133e094686bc547f9a33201cb4eb91a9